Drive/rover_drive_system.hpp

```diff
@@ -339,6 +339,18 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
       throw e;
     }
   }
+  double calculateLeftTurnAngle(double x_angle)
```

Comment:
```suggestion
  double GetLeftWheelDriveAngle(double angle)
```
x_angle --> angle is just more concise/cleaner